requirements.txt

```diff
@@ -1,5 +1,5 @@
 pyrovision@git+https://github.com/pyronear/pyro-vision.git#egg=pyrovision
-python-dotenv >= 0.15.0
-requests>=2.25.1
 pyroclient@git+https://github.com/pyronear/pyro-api.git#egg=pyroclient&subdirectory=client
-psutil
+python-dotenv == 0.15.0
```

Comment:
can you confirm it can accept spaces around the double equal sign?

Reply:
it seems to work but I will remove it just in case